aws_benchmark/crucible_simple_test_vpc/helios_user_data.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
```

Comment:
Unfortunately AWS does not terminate an instance if the user data script fails, so exiting doesn't tell us anything. By doing the `touch /var/booted_ok` at the end we can do this check with Ansible after the instance has booted.

There is some overlap here with what the helios-engvm repo's `gen_userdata.sh` does but I'm not sure how much of it can be pulled into an AMI while still being generic.